src/sentry/utils/snuba.py

```diff
@@ -377,3 +376,15 @@ def get_related_project_ids(column, ids):
                 project_field + '__isnull': False,
             }).values_list(project_field, flat=True)
     return []
+
+
+def insert_raw(data):
+    data = json.dumps(data)
+    try:
+        with timer('snuba_insert_raw'):
+            return _snuba_pool.urlopen(
+                'POST', '/tests/insert',
```

Comment:
So are we assuming that this eventstream backend is only used for local/test environment, given the limitations of this `tests/insert` endpoint to only insert in the `test` table?

Reply:
As of now, yes. The primary audience for this is local sentry devs.

We can figure out what this might look like as a production mode, without kafka, but I don't think that's too useful, since I think most likely, we'd just use Redis instead.

So probably this whole thing is temporary-ish until we get more pieces in place.

Reply:
SGTM